src/runtime/virtcontainers/kata_agent.go

```diff
@@ -177,6 +181,32 @@ func getSandboxPath(id string) string {
 	return filepath.Join(kataHostSharedDir(), id)
 }
 
+// Use in nydus case, guest shared dir is compatible with virtiofsd sharedir
+// nydus images are presented in kataGuestNydusImageDir
+//
+// virtiofs mountpoint: "/run/kata-containers/shared/"
+// kataGuestSharedDir: "/run/kata-containers/shared/containers"
+// kataGuestNydusImageDir: "/run/kata-containers/shared/images"
+var kataGuestNydusRootDir = func() string {
+	if rootless.IsRootless() {
+		// filepath.Join removes trailing slashes, but it is necessary for mounting
+		return filepath.Join(rootless.GetRootlessDir(), defaultKataGuestNydusRootDir) + "/"
+	}
+	return defaultKataGuestNydusRootDir
+}
+
+var imageMountPath = func(cid string) string {
+	return filepath.Join("/images", cid, lowerDir)
```

Comment:
`images` is a too common name, how about changing to `nydus`? And this directory will contain volumes too?

Reply:
I think `images` will be more suitable, as this directory only containes rafs(image rootfs),  maybe `images` will be use in other cases？

Reply:
+1 for `nydus` or `rafs` as `images` is too generic. Also, `nydus` and `rafs` are shorter than `images` ;) We have hit path limits before so it's not just bike-shedding :smile:

Reply:
ok, I have changed to `rafs`